### PR TITLE
Add support for radios endpoint and fix Tracks#from-external get requests

### PR DESCRIPTION
**Pull request by fermaud**

### CODE REVIEW

**Comment by mehlah** [comment]
@gcolson @fermaud can you pls rebase this one on master?
Looks like something conflicts.

**Comment by gcolson** [comment]
Yep sorry i didn't see your comment.. Should be ok now

File: lib/siilar/client/clients.rb [hunk 1]
```diff
@@ -13,9 +13,12 @@ def tags
       @services[:tags] ||= Client::TagsService.new(self)
     end
 
+    def tracks
+      @services[:tracks] ||= Client::TracksService.new(self)
+    end
+
     def users
       @services[:users] ||= Client::UsersService.new(self)
-    end
```
Comment (mehlah) [inline lib/siilar/client/clients.rb]: Looks like this guy is the guilty one :cop: 
https://travis-ci.org/craftsmen/niland-siilar-ruby/builds/102544831
@gcolson could you fix it pls?

Comment (gcolson) [inline lib/siilar/client/clients.rb]: done

File: lib/siilar/client/clients.rb [hunk 2]
```diff
@@ -42,6 +46,11 @@ class TagsService < ClientService
 
     class UsersService < ClientService
       include Client::Users
+
```
Comment (mehlah) [inline lib/siilar/client/clients.rb]: @gcolson The build still fails.
Looks like and `end` is missing here too for this class.

**Comment by mehlah** [comment]
LGTM
thanks @gcolson

**Pull request merged**